metadata/metadata_test.go

```diff
@@ -235,3 +245,18 @@ func BenchmarkJSONMetadata_UnmarshalJSON(b *testing.B) {
 		}
 	}
 }
+
+func BenchmarkMarshalJSON(b *testing.B) {
```

Comment:
Any stats available to showoff? :nerd_face:

Reply:
Yep. 
Started with:
```
BenchmarkMarshalJSON-4            300000              4309 ns/op            1160 B/op         18 allocs/op
```
Ended with:
```
BenchmarkMarshalJSON-4     	 1000000	      1764 ns/op	     312 B/op	       5 allocs/op
```